temporal-sdk/src/test/java/io/temporal/workflow/childWorkflowTests/ChildWorkflowWithCronScheduleTest.java

```diff
@@ -56,11 +56,7 @@ public void testChildWorkflowWithCronSchedule() {
         testWorkflowRule
             .getWorkflowClient()
             .newUntypedWorkflowStub(
-                "TestWorkflow1",
-                newWorkflowOptionsWithTimeouts(testWorkflowRule.getTaskQueue())
-                    .toBuilder()
-                    .setWorkflowRunTimeout(Duration.ofHours(10))
```

Comment:
Removed because it's already set in the newWorkflowOptionsWithTimeouts and this timeout doesn't matter anyway